src/Grid/utils.jl

```diff
@@ -340,3 +340,38 @@ for (func,                   entity_f,            entity_t,     filter_f,
     end
 end
 end
+
+# Hypercube is simply ⨂ᵈⁱᵐ Line :)
+sample_random_point(::Type{Ferrite.RefHypercube{ref_dim}}) where {ref_dim} = Vec{ref_dim}(ntuple(_ -> 2.0*rand()-1.0, ref_dim))
+
+# Dirichlet type sampling
+#
+# The idea behind this sampling is that the d-Simplex (i.e. a generalized triangle in d dimensions)
+# is just a surface in d+1 dimensions, that can be characterized by two constraints:
+# 1. All coordinates are between 0 and 1
+# 2. The sum of all coordinates is exactly 1
+# This way we can just sample from the d+1 dimensional hypercube, transform the hypercube
+# logarithmically to get a "normal distribution" over our simplex and enforce that the coordinates
+# sum to 1. By dropping the last coordinate in this sample we finally obtain d numbers which lies in
+# the d-simplex.
+#
+# A nice geometric sketch of this process is given in this stackexchange post: https://stats.stackexchange.com/a/296779
+function sample_random_point(::Type{Ferrite.RefSimplex{ref_dim}}) where {ref_dim} # Note that "ref_dim = d" in the text above
+    ξₜ = ntuple(_ -> -log(rand()), ref_dim+1)
+    return Vec{ref_dim}(ntuple(i->ξₜ[i], ref_dim) ./ sum(ξₜ))
+end
+
+# Wedge = Triangle ⊗ Line
+function sample_random_point(::Type{RefPrism})
+    (ξ₁, ξ₂) = sample_random_point(RefTriangle)
+    ξ₃ = rand(Float64)
+    return Vec{3}((ξ₁, ξ₂, ξ₃))
+end
+
+# TODO what to do here? The samplig is not uniform...
+function sample_random_point(::Type{RefPyramid})
+    ξ₃ = (1-1e-3)*rand(Float64) # Derivative is discontinuous at the top
+    # If we fix a z coordinate we get a Quad with extends (1-ξ₃)
+    (ξ₁, ξ₂) = (1.0 - ξ₃) .* Vec{2}(ntuple(_ -> rand(), 2))
+    return Vec{3}((ξ₁, ξ₂, ξ₃))
+end
```

Comment:
Are these used for anything but testing?

Reply:
I use them downstream for testing, too, and thought I should just expose these here rather than copy paste them into the downstream packages.

If we don't want to have such functions in core we can start discussing about something like FerriteUtils.jl?

Reply:
I see the need - but I'm just worried that it is a slippery slope adding convenience functions not used will bloat the core. A test dep utility package also adds to the maintenance load, but at least reduces the core package. Not sure what the best option is for these cases...